### PR TITLE
add missing dist files

### PR DESCRIPTION
**Pull request by TomWoodward**
[CORE-525]
https://github.com/openstax/h5p-runner/pull/6

[CORE-525]: https://openstax.atlassian.net/browse/CORE-525?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by RoyEJohnson — COMMENTED** [review]
This is fine, but doesn't change the text color for dialogcards-progress.

**Comment by TomWoodward** [comment]
yea, we should add that to the 1.8, but this is just making sure the dists don't get lost

File: libraries/H5P.Dialogcards-1.8/dist/h5p-dialogcards.css [hunk 1]
```diff
@@ -0,0 +1 @@
...ent:"\F01E"}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table{width:100%}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td{height:1.6em;border-bottom:1px solid #d9d9d9;text-align:left}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-category{width:100%}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-symbol{padding-left:.5em}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-symbol.h5p-dialogcards-check:before{font-family:H5PFontAwesome4;content:"\F00C";color:#1f824c}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-symbol.h5p-dialogcards-times:before{font-family:H5PFontAwesome4;content:"\F00D";color:#d95354}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-score{padding-left:.5em;text-align:right;font-weight:700}.h5p-dialogcards-summary-screen .h5p-dialogcards-summary-table td.h5p-dialogcards-summary-table-row-score-divider{color:#757575;font-size:.75em}.h5p-dialogcards-gone{display:none;visibility:hidden}
```
Comment (RoyEJohnson) [inline libraries/H5P.Dialogcards-1.8/dist/h5p-dialogcards.css:1]: This is packed, but note `.h5p-dialogcards-progress{color:#999` will revert to the non-accessible color.

**Comment by RoyEJohnson** [comment]
Oh, I didn't notice you were doing 1.8. 😳

**Comment by TomWoodward** [comment]
yea this was adding the files that were missing from the repo, so we don't accidentally delete them

**Pull request merged**